media-libs/vulkan-loader/vulkan-loader-1.2.179.ebuild

```diff
@@ -36,6 +35,10 @@ DEPEND="${PYTHON_DEPS}
 "
 PDEPEND="layers? ( media-libs/vulkan-layers:=[${MULTILIB_USEDEP}] )"
 
+PATCHES=(
+	"${FILESDIR}"/CHROMIUM-Fix-cross-compilation.patch
```

Comment:
Missing?

Reply:
Oops. Didn't mean to leave that hunk in.